### ISSUE TITLE
README.md 'Training New Models' steps are not working

### ISSUE THREAD

**Issue by lovato** [RasaHQ/rasa_nlu#622]
**Rasa NLU version** (e.g. `0.7.3`):
0.10.0 w/ PIP
**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):
None
**Operating system** (windows, osx, ...):
Linux
**Issue**:

After trying to train a new model (json or MD), I got:
`{"error": "Missing project name to train"}`

On the code, it indeed expects for a query parameter called "project".
I tried to send it, and it looks going a step further, but w/o lucky.

**Content of configuration file** (if used & relevant):
```json

```

**Comment by tmbo** [comment]
Thanks for noting that. the curl request wasn't quite right. I updated that. Let me know if there are still issues.

**Comment by lovato** [comment]
@tmbo , tks. I got over this training step for json. Unfortunately, things are not working yet.

1) You fixed the line for json, but not sure if for MD, is actually "train?name" or needs to be "train?project" like you just did for json? They are different.

2) I used JSON. It worked. But I simply cannot make it to work from here. I keep getting:
`WARNING:rasa_nlu.project:Invalid model requested. Using default`
even with this status:
`{"available_projects": {"default": {"status": "ready", "available_models": ["fallback"]}, "test_model": {"status": "ready", "available_models": ["model_20171006-142730", "fallback"]}}}`

Best,
